### PR TITLE
Issue58 deobfuscate constants

### LINKED ISSUE #58: Deobfuscate and document constants in defs.h
Most important is to deobfuscate the constants that are (also) used outside SDP4/SGP4, since what happens only within SDP4/SGP4 might be total madness anyway. In any case, constants should at least be in upper-case. 

[Spacetrack report #3](https://celestrak.com/NORAD/documentation/spacetrk.pdf) might help in finding out what the different constants actually are.

### PR DESCRIPTION
**Pull request by bjorgan**
Fix #58.

### CODE REVIEW

File: src/defs.h [hunk 1]
```diff
@@ -10,64 +10,142 @@
...
-#define thdt           4.3752691E-3
-#define rho            1.5696615E-1
-#define mfactor                7.292115E-5
-#define sr             6.96000E5       /* Solar radius - km (IAU 76) */
-#define AU             1.49597870691E8 /* Astronomical unit - km (IAU 76) */
+#include <float.h>
 
-/* doppler calculation */
-#define SPEED_OF_LIGHT 299792458.0
+/**
```
Comment (ryeng) [inline src/defs.h]: I suggest we stick to one Doxygen comment style. I also suggest the triple slash style.

See https://www.stack.nl/~dimitri/doxygen/manual/grouping.html#memgroup for how to group these constants and document the groups.

File: src/defs.h [hunk 2]
```diff
@@ -10,64 +10,142 @@
...
+ * Mathematical convenience constants used by sgp4/sdp4 and related routines
+ */
+// pi
```
Comment (ryeng) [inline src/defs.h]: Here, and elsewhere, use /// (triple slash) to get Doxygen comments.

Comment (bjorgan) [reply src/defs.h]: Fixed in ca56f55e2ac9c5ca1d290d7dcca42f89ad861766

File: src/defs.h (hunk 1 above)
Comment (bjorgan) [reply src/defs.h]: Fixed in 969fde5ee749ae7e09bd7cfc157bb441836c23b9

**Review by ryeng — APPROVED** [review]
LGTM

**Pull request merged**